docs/conf.py

```diff
@@ -92,9 +92,20 @@
 
 # -- Options for HTML output ---------------------------------------------------
 
+# https://github.com/snide/sphinx_rtd_theme#using-this-theme-locally-then-building-on-read-the-docs
+#
+# on_rtd is whether we are on readthedocs.org, this line of code grabbed from docs.readthedocs.org
+on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
```

Comment:
Why not always use local theme? We did wanted to tune it up a bit to have more of look and feel of community (obviously not identical layout)